src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -564,9 +557,9 @@ static unique_ptr<Scene> _useBuildHelper(const SvgNode* node, float vx, float vy
 }
 
 
-static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, float vx, float vy, float vw, float vh, const string& svgPath)
+static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, float vx, float vy, float vw, float vh, const string& svgPath, bool mask)
 {
-    if (_isGroupType(node->type)) {
+    if (_isGroupType(node->type) || mask) {
```

Comment:
How about adding `SvgNodeType::Mask` to the if condition of _isGroupType?
I haven't tested that method yet. Could you please check it?

Reply:
It won't work. I need to go inside this 'if condition' for the 'SvgNodeType::Mask' only if the _sceneBuildHelper was called from `applyComposite`.

Reply:
Ok i got it. thank you for check :)